telemetry/roles/grafana_config/tasks/add_dashboards.yml

```diff
@@ -20,6 +20,7 @@
     grafana_password: "{{ grafana_password }}"
     title: "{{ telemetry_folder_name }}"
     state: present
+    use_proxy: false
```

Comment:
Shouldn't this be a placeholder that applies to all or none?

Reply:
This applies to all and working as expected in all cases